docker-compose.yaml

```diff
@@ -27,7 +27,7 @@ services:
         build:
             context: ./backend
             dockerfile: DockerfileDevelop
-            target: base
+            target: base # Set to debug to use debugpy
```

Comment:
This is not ideal. Can't we make a script for this that would do something like
```
cd backend && docker run debug python -m debugpy ...
```
and leave docker-compose unchanged? Or would this mean we wouldn't get the necessary communication between the Django server and the debugpy server?

Reply:
Yeah, good point. I'll try and see if a better solution can be found for this in the form of a script or dedicated environment variable.